2025/week14/solutions/Piece_correction.py

```diff
@@ -0,0 +1,24 @@
+import random
+
+
+# La fonction Piece retourne une liste contenant des 0 et des 1, considérez un 1 comme un succès, i.e. une fois où la pièce tombe sur pile, et 0 comme un échec
+def Piece(l):
+    return [random.randint(0, 1) for i in range(l)]
+
+
+def proba(n, l, iter=10000):
+    # n correspond au nombre de succès et l au nombre d'essais. Iter correspond au nombre d'expérience que vous allez
+    # réaliser pour obtenir la réponse. Cela devrait être grand mais pas trop (sinon le programme prendra trop de
+    # temps à s'exécuter). 10000 est un bon nombre d'itérations.
+    occurences = 0
+    for i in range(iter):
+        temp = Piece(l)  # On simule une expérience de l lancers.

```

Comment:
Same here, I would not keep `temp`
```suggestion
        flips = Flip(l)  # On simule une expérience de l lancers.
```